services/apps/test/apps/persistence/categories_test.clj

```diff
@@ -1,7 +1,14 @@
 (ns apps.persistence.categories-test
   (:use [clojure.test]
         [apps.persistence.categories]
-        [apps.test-db-setup]))
+        [apps.test-fixtures :only [run-integration-tests with-test-db]]
+        [korma.core]))
+
+(defn clean-up-hierarchy-versions [f]
+  (f)
+  (delete :app_hierarchy_version))
+
+(use-fixtures :each with-test-db run-integration-tests clean-up-hierarchy-versions)
```

Comment:
You could also use `:once` for the `run-integration-tests` and `with-test-db` fixtures:
```clojure
(use-fixtures :once run-integration-tests with-test-db)
```
That way you could ensure the fixtures run before every test, and only need to run once for the entire test run.
I did this for the metadata integration tests:
https://github.com/cyverse/DE/blob/dev/services/metadata/test/integration_tests.clj
https://github.com/cyverse/DE/blob/dev/services/metadata/test/integration_tests/metadata/services/avus_test.clj#L21